<launch>
  <!-- mandatory parameters -->
  <arg name="robosem_bridge_name"             default="ROCONBRIDGE"/> <!-- Name of ROCON Application -->
  <arg name="robosem_ip"                        default="192.168.10.111"/> <!-- IP of Robosem -->
  <arg name="robosem_port"                     default="6001"/> <!-- Prt of Robosem, It must not been chaged. -->

  <node pkg="robosem_bridge" type="robosem_bridge.py" name="robosem_bridge">
    <!-- parameters -->
    <param name="robosem_bridge_name"             value="$(arg robosem_bridge_name)"/>
    <param name="robosem_ip"                        value="$(arg robosem_ip)"/>
    <param name="robosem_port"                      value="$(arg robosem_port)"/>
  </node>
</launch>
